### DISCUSSION TITLE
Error with merging records in CCH2

### DISCUSSION THREAD

**Discussion by rcrowe-IRVC** [Symbiota/community-board#539] [Problems/Bugs]
Hi all - We are getting an error message and unexpected results when trying to merge records using the linking accession numbers and barcodes workflow.
Here is the error message:
_"This page isn't working
cch2.org is currently unable to handle this request
HTTP ERROR 500"_

The unexpected results are that the image is getting copied over to the record holding the existing data (the record with the accession number/ other cat number), so that there are now two records with the same image.

In the meantime, I am planing to use the copy all fields button to copy transcribed data to records with new specimen images, then delete the image with only the data. We have around 3,400 unprocessed records, it would be great to get the merge function working again to continue chipping away at these.

Thanks!

**Comment by themerekat** [comment]
Hi @rcrowe-IRVC, this has been reported and we are currently working to fix it!

**Reply by rcrowe-IRVC** [reply]
Thanks! I'll keep tabs on the issue in the link. I promise I did a search for the issue before I posted :)

**Reply by themerekat** [reply]
No worries! Our system is a little confusing...we have 2 repositories going at the same time. This one consists of all of our wish list items: https://github.com/BioKIC/symbiota-docs and this one consists of all of our bugs in the works: https://github.com/BioKIC/Symbiota

**Reply by themerekat** [reply]
(I'm also going to reopen for now, until the problem is fixed!)

**Comment by themerekat** [comment]
https://github.com/BioKIC/Symbiota/issues/1327
